packages/model/src/tests/workspace.test.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Workspace,
+  WorkspaceOptions,
+  ProjectKind,
+  Service,
+  WebApp,
+  Package,
+} from '..';
+import {Base, Options} from '../base';
+
+// Test creation of different types of workspaces
+describe('Workspace', () => {
+  const name = 'test';
+  const root = '../../';
+
+  const options: Options = {
+    name,
+    root,
+  };
+  const base = new Base(options);
```

Comment:
There's a lot jumbled up here and it's hard to follow. I think we can break these up into their own respective `describe` blocks. 

```tsx
describe('Workspace', () => {
  describe('package', () => {});
  describe('webApp', () => {});
  describe('service', () => {});
});
```

Reply:
I have since created some utilities which cleans up these tests, I will update and call for review again after I have addressed the other comments as well.